.github/release.yml

```diff
@@ -0,0 +1,29 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Install Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: 14.x
+    - run: npm install
+    - name: Publish package 
+      run: npx vsce publish
+      env: 
+        VSCE_TOKEN: ${{ secrets.VSCE_TOKEN }}
+    - name: Patch Version
+      run: npm version patch 
```

Comment:
Q: Mit `npx vsce publish` und `npm version patch` werden die Versionen in [.vsixmanifest](https://github.com/GeoWerkstatt/vsc_interlis2_extension/blob/master/.vsixmanifest#L6), [package.json](https://github.com/GeoWerkstatt/vsc_interlis2_extension/blob/master/package.json#L11) und [package-lock.json](https://github.com/GeoWerkstatt/vsc_interlis2_extension/blob/master/package-lock.json#L3) magisch auf eine neue identische Version gebracht? Ich frage deshalb, weil die Versionen in den aufgeführten Files aktuell unterschiedlich sind.

Reply:
Die im .vsxmanifest werden damit nicht gepatcht. Ich weiss auch nicht genau welchen Einfluss diese Version überhaupt hat.

Reply:
Wenn ich das richtig sehe, ist .vsixmanifest [inzwischen obsolet](https://code.visualstudio.com/api/references/extension-manifest) und sollete wohl in [package.json](https://github.com/GeoWerkstatt/vsc_interlis2_extension/blob/master/package.json) integriert werden. @flenny Was meinst du dazu?

Reply:
> Q: Im Repo gibts noch die [CHANGELOG.md](https://github.com/GeoWerkstatt/vsc_interlis2_extension/blob/master/CHANGELOG.md). Müsste die noch angepasst werden?

Der CHANGELOG muss mit dieser config nun vor der merge auf `master` korrekt geschrieben werden.

Reply:
> Wenn ich das richtig sehe, ist .vsixmanifest [inzwischen obsolet](https://code.visualstudio.com/api/references/extension-manifest) und sollete wohl in [package.json](https://github.com/GeoWerkstatt/vsc_interlis2_extension/blob/master/package.json) integriert werden. @flenny Was meinst du dazu?

Wunderbar 🎉 Das macht das ganze sehr viel klarer! 👍🏻